### PR TITLE
Add README files to explain AI samples

### PR DESCRIPTION
**Pull request by louie-tsai**
## Description

Add README files to explain AI samples.

## External Dependencies

None.

## Type of change

Please delete options that are not relevant. Add a 'X' to the one that is applicable. 

- [X ] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

NA

## Checklist for New samples:
NA

### CODE REVIEW

**Comment by louie-tsai** [comment]
@JoeOster 
Could you help to review this PR?
thanks

File: AI-and-Analytics/Features-and-Functionality/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+## License
+Code samples are licensed under the MIT license. See
+[License.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/License.txt) for details.
+
+Third party program Licenses can be found here: [third-party-programs.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/third-party-programs.txt)
+
+# Features and Functionalities Samples
+
+| Compoment      | Folder                                             | Description
+| --------- | ------------------------------------------------ | -
```
Comment (JoeOster) [inline AI-and-Analytics/Features-and-Functionality/README.md:18]: Please organize this by the first column in Alpha order

File: AI-and-Analytics/Getting-Started-Samples/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+## License
+Code samples are licensed under the MIT license. See
+[License.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/License.txt) for details.
+
+Third party program Licenses can be found here: [third-party-programs.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/third-party-programs.txt)
+
+# Getting Started Samples
+
+| Compoment      | Folder                                             | Description
```
Comment (JoeOster) [inline AI-and-Analytics/Getting-Started-Samples/README.md:17]: Please organize this by the first column in Alpha order

**Review by JoeOster — CHANGES_REQUESTED** [review]

File: AI-and-Analytics/Getting-Started-Samples/README.md (hunk 2 above)
Comment (louie-tsai) [reply AI-and-Analytics/Getting-Started-Samples/README.md:17]: done. thanks.

**Pull request merged**